bitrise.yml

```diff
@@ -55,6 +91,10 @@ workflows:
                 echo "Does not exist: apk file's path"
                 exit 1
             fi
+    - change-workdir:
+        title: Change back workdir
+        inputs:
+        - path: ..
```

Comment:
instead of magic relative path, pls use this kind of approach: https://github.com/bitrise-steplib/steps-npm/blob/master/bitrise.yml#L7